src/common/addlayers/RegistryLayersDirective.js

```diff
@@ -269,6 +278,15 @@
 
             // load saved registry layers if they exist
             scope.addRegistryLayersFromSavedMap();
+
+            function cartLayer(layerConfig) {
```

Comment:
`cartLayer` is not a very good name for this method. 
Does it reset the previewLayers to the layerConfig that gets passed into it, correct?
Then a name might be `resetPreviewLayersWithConfig`